core/src/main/java/discord4j/core/event/domain/channel/PrivateChannelCreateEvent.java

```diff
@@ -26,7 +26,10 @@
  * This event is dispatched by Discord.
  *
  * @see <a href="https://discord.com/developers/docs/topics/gateway#channel-create">Channel Create</a>
+ *
+ * @deprecated This event will no longer be dispatched by Discord in v8 and thus, will no longer be present in 3.2.
```

Comment:
I think we should link to the changelog here as well.

Reply:
Done !